.gitignore

```diff
@@ -25,4 +25,11 @@ libtool
 isa-l.h
 /libisal.la
 libisal.pc
+crc/*_perf
+erasure_code/*_perf
+erasure_code/*_test
+erasure_code/gf_vect_dot_prod_1tbl
+igzip/*_perf
+mem/*_perf
 programs/igzip
+raid/*_perf
```

Comment:
Could you add *_test for crc, igzip, mem and raid? Thanks.

Reply:
Added, thanks :)